hashtagmap-admin/src/main/java/com/songpapeople/hashtagmap/blacklist/service/dto/BlackListResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.songpapeople.hashtagmap.blacklist.service.dto;
+
+import com.songpapeople.hashtagmap.instagram.domain.model.Instagram;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Getter
+public class BlackListResponse {
+    private Long placeId;
+    private String replaceName;
+    private Long hashtagCount;
+
+    private BlackListResponse(Long placeId, String replaceName, Long hashtagCount) {
+        this.placeId = placeId;
+        this.replaceName = replaceName;
+        this.hashtagCount = hashtagCount;
+    }
+
+    public static BlackListResponse of(Instagram instagram) {
+        return new BlackListResponse(instagram.getPlaceId()
+                , instagram.getHashtagName()
+                , instagram.getHashtagCount());
+    }
```

Comment:
```suggestion
    public static BlackListResponse of(Instagram instagram) {
        return new BlackListResponse(instagram.getPlaceId()
                , instagram.getHashtagName()
                , instagram.getHashtagCount());
    }

    private BlackListResponse(Long placeId, String replaceName, Long hashtagCount) {
        this.placeId = placeId;
        this.replaceName = replaceName;
        this.hashtagCount = hashtagCount;
    }
```

static function이 위로 올라오도록 해주세요~